agents/argument_parser.py

```diff
@@ -100,8 +100,12 @@ def add_mc_parser(self):
         )
         mc_parser.add_argument(
             "--draw_map", 
-            default="memory", 
```

Comment:
locobot_agent.py also uses this arg, so that has to be updated as well: https://github.com/facebookresearch/fairo/blob/2e00b7b42519e27cfaebd25cec3c6327b79759df/agents/locobot/locobot_agent.py#L268-L273

Reply:
maybe a better way would be to keep this option as-is but rename it `map_data`.
Then, also have a flag `draw_map` that either enables it or not.

Reply:
> maybe a better way would be to keep this option as-is but rename it `map_data`. Then, also have a flag `draw_map` that either enables it or not.

Ah ok, that makes sense to break it down. Thanks Soumith!